### PR TITLE
Add summer 2020 newsletter

### PR DESCRIPTION
**Pull request by ElenaFdR**
This is the same text that was sent to newsletter subscribers in August 2020.

-----
[View rendered _posts/2020-12-02-summer-2020-newsletter.md](https://github.com/publiccodenet/blog/blob/summer-2020-newsletter/_posts/2020-12-02-summer-2020-newsletter.md)

### CODE REVIEW

File: _posts/2020-12-02-summer-2020-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Welcome to our newsletter, bumper summer edition. Like last summer, many of the people we work with went on holiday, but that hasn't slowed us down.
+
+🏅 Perfectly timed to get us pumped for summer, we got an Encouragement Award from the Netherlands Internet Society (as part of their annual Innovation Award) in June. [Read more on our blog](https://blog.publiccode.net/news/2020/06/17/isoc-encouragement-award-consider-us-encouraged.html).
+
+## What we've been up to 👩‍💻 🧑🏻‍🤝‍🧑🏽 🤙
+
+### Codebase stewardship
+
+* We currently have 2 codebases under stewardship: [Signalen](https://publiccode.net/codebases/signalen.html) and [OpenZaak](https://publiccode.net/codebases/openzaak.html). Both codebases are focused on growing their open source communities in 2020.
+* In June, we built a prototype product marketing website fo OpenZaak: <openzaak.org>.
```
Comment (Ainali) [inline _posts/2020-12-02-summer-2020-newsletter.md:25]: Should this be a link to openzaak.org?

File: _posts/2020-12-02-summer-2020-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* We reorganized how the Board of Directors gets updates from the Foundation. [Ben](https://publiccode.net/team/ben-cerveny.html), [Boris](https://publiccode.net/team/boris-van-hoytema.html), [Laura](https://publiccode.net/team/laura-scheske.html) and [Claus](https://publiccode.net/team/claus-mullie.html) are now working on a logframe and roadmap so there's more clarity and predictabilty about our goals and how we'll get there. We'll publish these when they're ready.
+* In operations, [Deborah](https://publiccode.net/team/deborah-meibergen.html) has updated our financial forecasting model, upgraded our chat server, launched new tools to support our community (Mailman, Podbean and StreamYard), and has been experimenting with putting Jitsi on a more stable and scalable footing.
+* For communications, [Floris](http://flrs.nl/contact) is producing beautiful new illustrations and we've reorganised the [illustrations repo](https://github.com/publiccodenet/illustrations) so each illustration has better metadata. [Elena](https://publiccode.net/team/elena-findley-de-regt.html) and [Ryan](http://www.angelplasma.net/work/) are working on a refresh of <publiccode.net>.
```
Comment (Ainali) [inline _posts/2020-12-02-summer-2020-newsletter.md:42]: Should the last word be linked to publiccode.net?

**Review by Ainali — APPROVED** [review]
I am not sure if these need to be links, but I think that could be beneficial (but it's not a blocker).

File: _posts/2020-12-02-summer-2020-newsletter.md (hunk 1 above)
Comment (ElenaFdR) [reply _posts/2020-12-02-summer-2020-newsletter.md:25]: Yup! In other markdown, this generates a link.

**Pull request merged**